app/assets/stylesheets/layout_components/buttons/button-grouping.scss

```diff
@@ -153,6 +152,10 @@
     &--blue {
       color: $brand-blue-light;
     }
+
+    &--charcoal {
+      color: $charcoal;
+    }
```

Comment:
Hey Sinead, what's the reason for using `charcoal` as the text colour in these buttons, and not `slate` like we use for the majority of our text?

Reply:
lol cause I screwed up

Reply:
looks good!